### DISCUSSION TITLE
Percentage colors

### DISCUSSION THREAD

**Discussion by ShadowAce** [fastfetch-cli/fastfetch#687] [Q&A] [answered]
First of all, I'd like to thank the developer(s) for this wonderful product.  It has been VERY useful!

I've been searching all over the place and cannot seem to find the answer.  Perhaps it's hardcoded?

I would like to change the percentage colors shown--particularly on disk usage.  Our organization monitors our servers based on disk usage, and we try to take action when usage hits 85%, though that can vary, depending on the server.

I like the green when usage is low, but would like to change it to red when usage hits 85%, and yellow somewhere in between.

I have not been able to find it in a config file, nor anywhere online where someone has done this before.

Any suggestions?

**Comment by CarterLi** [comment, marked as answer]
```jsonc
{
    "type": "disk",
    "percent": {
        "green": 20, // Shows result in green when the usage is less than 20%
        "yellow": 50 // Shows result in yellow when the usage is between 20% and 50%
                     // Shows result in red when the usage is greater than 50%
    }
}
```

![image](https://github.com/fastfetch-cli/fastfetch/assets/6134068/efdb1ad5-9b60-4e07-9479-f08230621795)

**Comment by ShadowAce** [comment]
Awesome!  We absolutely appreciate this--and there is NO WAY we can get this kind of response time from a commercial package.

Thank you!

**Reply by CarterLi** [reply]
If like it, please recommand it to others.

Thanks.

**Comment by ShadowAce** [comment]
Yes--I have been and will continue to do so.
